BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeElementUITextField.swift

```diff
@@ -8,7 +8,7 @@
 import UIKit
 
 final public class CardVerificationCodeElementUITextField: TextElementUITextField {
-    override var validation: ((String?) -> Bool)? {
+    public override var validation: ((String?) -> Bool)? {
```

Comment:
allowing `validation` to be publicly set means the user can see the plaintext value that’s set within a textfield since we pass in the plaintext value as the arg for that function. we need to have a discussion as a bigger group before allowing this

```suggestion
    override var validation: ((String?) -> Bool)? {
```

Reply:
Ah good point! I thought it was just missing. 
Initially I thought you didn't need to add `public` so when I couldn't access the mask I thought that was a mistake.
Reverted!